### PR TITLE
Separate test support in to project

### PR DESCRIPTION
**Pull request by JGulbronson**
This should allow https://github.com/PagerDuty/rt-idempotency/pull/33 to be merged, which is hopefully the last step for Scheduler to cross-compile.

### CODE REVIEW

**Comment by JGulbronson** [comment]
The 2nd commit will be the potentially more controversial one. I moved the library version in to `build.sbt` as a common setting, so that it's shared between the three projects. It would feel odd to me to have the the projects at different versions, for most cases, because they're still very dependent on each other. This would also allow projects that use this as a dependency to have a variable shared between the projects it imports, something like:
```sbt
libraryDependencies ++= Seq(
    "com.pagerduty" %% "eris-core" % erisCoreVersion,
    "com.pagerduty" %% "eris-core-test-support" % erisCoreVersion
)
```

File: common/src/main/scala/com/pagerduty/eris/ClusterCtx.scala [hunk 1]
```diff
@@ -51,6 +51,7 @@ import com.netflix.astyanax.{ AstyanaxContext, Cluster }
  * val cluster = clusterCtx.cluster
  * }}}
  */
+
```
Comment (DWvanGeest) [inline common/src/main/scala/com/pagerduty/eris/ClusterCtx.scala]: Remove whitespace change.

File: version.sbt [hunk 2]
```diff
@@ -1 +0,0 @@
-version := "2.0.2"
```
Comment (DWvanGeest) [inline version.sbt]: It would be nice if we could keep this, but I do agree that all the libs should be versioned together. Is it possible to do both?

Comment (JGulbronson) [inline version.sbt]: I don't think there's a way to "inherit" the version, for lack of a better word. Having it in `build.sbt` is the most DRY way I could get it.

Comment (DWvanGeest) [inline version.sbt]: k, WFM.

**Pull request merged**